File: Unidad 2/Programa10.cs
using System;

namespace Programa10 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int Perimetro, Apotema;
            float multiplicacion;
            Console.WriteLine("Asigna el Perimetro: ");
            Perimetro = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Asigna la Apotema: ");
            Apotema = Convert.ToInt32(Console.ReadLine());
            multiplicacion = (Perimetro * Apotema) / 2;

            Console.WriteLine("El area del hexagono es: " + multiplicacion);
        }
    }
}

File: Unidad 2/Programa10.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa10;
import java.util.Scanner;
/**
 *
 * @author DELL
 */
public class Programa10 {

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int p, ap, area;
        System.out.print("Escriba el valor del perimetro: ");
        p = sc.nextInt ();
        System.out.print("Escriba el valor del apotema: ");
        ap = sc.nextInt ();
        area = (p * ap) / 2;
        System.out.print("El valor del area del hexagono es: " + area);
    }
}
